demos/shared/insurance_claims_service/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# took inspiration from https://medium.com/@albertazzir/blazing-fast-python-docker-builds-with-poetry-a78a66f5aed0
```

Comment:
Please remove this altogether. No need to keep this insurance_claims_service. We have a separate demo for that

Reply:
removed